src/app/routes/homePage/getInitialData/page-config.ts

```diff
@@ -0,0 +1,10 @@
+type Envs = 'test' | 'live';
+
+const HOME_PAGE_CONFIG: { [key: string]: { [key in Envs]: string } } = {
```

Comment:
**Future:** ensure that key is one of our defined services 
```suggestion
const HOME_PAGE_CONFIG: { [key: Service]: { [key in Envs]: string } } = {
```

Reply:
Sweet! let's keep this open for the next part

Reply:
I've added a link to this comment in the JIRA ticket for this work: WSTEAMA-313